flink-table-store-core/src/main/java/org/apache/flink/table/store/file/FileStoreOptions.java

```diff
@@ -94,7 +94,7 @@
     public static final ConfigOption<Duration> SNAPSHOT_TIME_RETAINED =
             ConfigOptions.key("snapshot.time-retained")
                     .durationType()
-                    .defaultValue(Duration.ofDays(1))
+                    .defaultValue(Duration.ofHours(1))
```

Comment:
Why do we change this?

Reply:
We do not expose the middle snapshot out, every read is read the latest, there is no need to keep it for such a long time.
Keeping it too long will result in a lot of small files remaining on the DFS.